### PR TITLE
UIOR-1214: allow order template to hide Currency and exchange rate fields

### PR DESCRIPTION
**Pull request by alisher-epam**
- [UIOR-1214](https://folio-org.atlassian.net/browse/UIOR-1214) - allow order template to hide Currency and exchange rate fields
- Dependent PR - https://github.com/folio-org/stripes-acq-components/pull/755
## Purpose

- Define `<IfFieldVisible.js />` module in `stripes-acqusitions-component`
- Define `<VisibilityControl />` module in `stripes-acqusitions-component`
https://github.com/folio-org/stripes-acq-components/assets/116072773/661e9c12-40ea-4519-ac98-2cc9c41fca3d

## Pre-Merge Checklist
Before merging this PR, please go through the following list and take appropriate actions.
- [x] I've added appropriate record to the CHANGELOG.md
- Does this PR meet or exceed the expected quality standards?
  - [ ] Code coverage on new code is 80% or greater
  - [ ] Duplications on new code is 3% or less
  - [ ] There are no major code smells or security issues
- Does this introduce breaking changes?
  - [ ] If any API-related changes - okapi interfaces and permissions are reviewed/changed correspondingly
  - [ ] There are no breaking changes in this PR.

If there are breaking changes, please **STOP** and consider the following:

- What other modules will these changes impact?
- Do JIRAs exist to update the impacted modules?
  - [ ] If not, please create them
  - [ ] Do they contain the appropriate level of detail?  Which endpoints/schemas changed, etc.
  - [ ] Do they have all they appropriate links to blocked/related issues?
- Are the JIRAs under active development?
  - [ ] If not, contact the project's PO and make sure they're aware of the urgency.
- Do PRs exist for these changes?
  - [ ] If so, have they been approved?

Ideally all of the PRs involved in breaking changes would be merged in the same day to avoid breaking the folio-testing environment.  Communication is paramount if that is to be achieved, especially as the number of intermodule and inter-team dependencies increase.

While it's helpful for reviewers to help identify potential problems, ensuring that it's safe to merge is ultimately the responsibility of the PR assignee.

### CODE REVIEW

**Comment by alisher-epam** [comment]
@folio-org/acquisitions-ui @folio-org/fe-tl-reviewers please review this PR

**Review by OleksandrHladchenko1 — APPROVED** [review]

File: src/components/POLine/Cost/CostForm.js [hunk 1]
```diff
@@ -61,8 +61,8 @@ const CostForm = ({
   order,
   initialValues,
   change,
-  required = true,
-  hiddenFields = {},
+  required,
```
Comment (NikitaSedyx) [inline src/components/POLine/Cost/CostForm.js:64]: any reasons to make not required changes?

Comment (alisher-epam) [reply src/components/POLine/Cost/CostForm.js:64]: I have just moved it to default props below https://github.com/folio-org/ui-orders/pull/1551/files/2730fb43a84a33e693d7cdd07cf0d6b94923eb3c#diff-baf38169fc987a803638e5b81f7db7388631b04aa64859f2dd877b5850ce43f3R341

Comment (NikitaSedyx) [reply src/components/POLine/Cost/CostForm.js:64]: I understand, but what is the reason?

Comment (alisher-epam) [reply src/components/POLine/Cost/CostForm.js:64]: I didn't quite understand, what are you implying?

Comment (NikitaSedyx) [reply src/components/POLine/Cost/CostForm.js:64]: don't do changes if you don't need to change

Comment (alisher-epam) [reply src/components/POLine/Cost/CostForm.js:64]: I haven't changed. I have just refactored the code according to our codebase pattern.

File: src/components/POLine/Cost/CostView.js
Comment (usavkov-epam) [inline src/components/POLine/Cost/CostView.js:1]: I am unable to find the updates associated with the concealment of fields as stated in the ticket.

Comment (alisher-epam) [reply src/components/POLine/Cost/CostView.js:1]: I am passing hidden fields here https://github.com/folio-org/ui-orders/pull/1551/files#diff-baf38169fc987a803638e5b81f7db7388631b04aa64859f2dd877b5850ce43f3R214

**Review by NikitaSedyx — APPROVED** [review]

**Pull request merged**